packages/docs/src/components/TableOfContents.tsx

```diff
@@ -37,14 +41,23 @@ export const TableOfContentsList = ({ items, level, className = '' }: TableOfCon
 /**
  * The Desktop version of the table of contents
  */
-const TableOfContents = ({ items }: TableOfContentsProps) => {
+const TableOfContents = ({ items, title }: TableOfContentsProps) => {
   const level = 1;
   return items.length ? (
     <div className="c-table-of-contents">
       <h2 className="c-table-of-contents__heading ds-u-margin-y--0 ds-u-font-size--base">
         On this page{' '}
       </h2>
       <TableOfContentsList items={items} level={level} />
+
+      <h2 className="c-table-of-contents__heading ds-u-margin-y--0 ds-u-md-margin-top--6 ds-u-font-size--base">
+        Have ideas?{' '}
+      </h2>
+      <ul role="list" className="ds-c-list--bare ds-u-md-margin-y--2">
+        <li>
+          <a href="/not-in-sidebar/feedback">Propose a change</a>
+        </li>
+      </ul>
```

Comment:
similar to the `<TableOfContentsList>`, if this is the same between desktop and mobile, could this be pulled out into a reusable function so it only needs to be maintained once?

Reply:
I've done this in #2024, asking Sarah to remove the TOC changes in this PR.